docs/load-test-how-to-use.md

```diff
@@ -0,0 +1,115 @@
+    - Edit `SSHPublicKey` in `deployer.json` after setting up ssh.
+    - `go run ./cmd/ltctl deployment create`
+        - Limit operations of `deployment` to a single shell window.
+        - If the deployment gets stuck, [check if AWS credentials are expired, and add new ones.](https://community.mattermost.com/core/pl/weau31yyp38btddryjuxbsnh1r).
+        - [Terraform actions are idempotent](https://community.mattermost.com/core/pl/jtebkneah3futd1y7pj8y9nrqy), so you rarely have to [destroy](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md#destroy-the-current-deployment) the deployment, if things go wrong while creating resources.
+    - Once the deployment is successful, stdout will contain information on server addresses for app server, agent, coordinator, and Grafana deployments.
+        - Open the Mattermost URL in browser to confirm the app is working.
+        - At this point, you can check the server logs by [ssh-ing](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md#ssh-access-to-the-terraformed-hosts) into the app instance. Once there, open `/opt/mattermost/logs/mattermost.log`.
+    - Gearing up to start the load-test:
+        - Use `agents'` URL and Prometheus URL in the `coordinator.json` file generated [here](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md#copy-default-config).
+        - Change `ConnectionConfiguration` in the `config.json` generated [here](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md#copy-default-config-1).
+        - Configure `InstanceConfiguration` in the same `config.json` file (which, as mentioned earlier, populates the Mattermost server's database with required data for the load test). Note that a heavier config with `NumPosts` would take a very long time get populated. Please refer to the [NB](/#nb) section below to manually populate the database from a backup, in order to bypass 'data-generation.
+    
+    - Start the load test with `go run ./cmd/ltctl loadtest start`.
+        - Once a loadtest is running, its status can be checked with `go run ./cmd/ltctl loadtest status`.
+        - `ssh` into one of the agent machines, and `cat ~/mattermost-load-test-ng/logs/ltagent.log` to verify the load-test is working without errors.
+        - Open the Grafana deployment with the URL from `go run ./cmd/ltctl deployment info`. 
+        - It takes some time for the deployment to stabilize. When the loadtest tool connects the `MaxActiveUsers` count of users to the app, there might be a big count of HTTP 4xx errors during this time.
```

Comment:
Should this be `Until`? cc/ @cwarnermm For context, Carrie: the big count of 4xx errors happen from the beginning of the test until all the users have been connected (this is the ramp-up phase, and then all those users stay connected until the end of the test, without more users added).

```suggestion
        - It takes some time for the deployment to stabilize. Until the loadtest tool connects the `MaxActiveUsers` count of users to the app, there might be a big count of HTTP 4xx errors during this time.
```